lib/agent/agent.vala

```diff
@@ -1684,21 +1684,28 @@ namespace Frida.Agent {
 	public class ThreadSuspendMonitor : Object {
 		private ThreadSuspendFunc thread_resume;
 		private ThreadResumeFunc thread_suspend;
+		private TaskThreadsFunc task_threads;
```

Comment:
Nit: maybe every `task` aspect, like here, should go first of those three, as a task is conceptually higher level than a thread? OTOH it is a secondary aspect of the class it belongs to, so maybe not. I'm fine with either 😊